documentation/staging/content/userguide/managing-domains/model-in-image/auxiliary-images.md

```diff
@@ -58,214 +53,110 @@ from additional images.
-The `mountPath` field is the location of a directory in an auxiliary image, and
-is also the location in the main pod container (which will automatically contain
-a recursive copy of the auxiliary image directory). The `name` field is 
-arbitrary, and is in turn referenced by one or more auxiliary images that 
-are defined separately using
-[Auxiliary images](#auxiliary-images) configuration.
-
-For example:
-
-```
-  spec:
-    auxiliaryImageVolumes:
-    - name: auxiliaryImageVolume1
-      mountPath: /auxiliary
-```
-
-#### Auxiliary images
-
-One or more auxiliary images can be configured in a domain resource
-`serverPod.auxiliaryImage` array.
-Each array entry must define an `image` and `volume`
-where `image` is the name of an auxiliary image
-and the `volume` is the name of an [auxiliary image volume](#auxiliary-volumes-and-paths)
-as described previously.
-Optionally , you can also specify an `imagePullPolicy`,
-which defaults to `Always` if the `image` ends in `:latest` and to `IfNotPresent`,
+Beginning with operator version 4.0, one or more auxiliary images can be configured in a domain resource
+`configuration.model.auxiliaryImages` array.
+Each array entry must define an `image` which is the name of an auxiliary image.
+Optionally, you can set the `imagePullPolicy`,
+which defaults to `Always` if the `image` ends in `:latest` and `IfNotPresent`,
 otherwise.
-Also, optionally, you can customize
-the command that is used to merge (copy) the auxiliary image's files
-into the auxiliary image volume during pod startup (this is rarely 
-needed, see [Performing replaces instead of merges](#performing-replaces-instead-of-merges) for an example).
+If image pull secrets are required for pulling auxiliary images, then the secrets must be referenced using `domain.spec.imagePullSecrets`.
+
+Also, optionally, you can configure the [source locations](#source-locations) of the WebLogic Deploy Tooling model 
+and installation files in the auxiliary image using the `sourceModelHome` and `sourceWDTInstallHome` fields described 
+[below](#source-locations).
 For details 
 about each field, see the [schema](https://github.com/oracle/weblogic-kubernetes-operator/blob/main/documentation/domains/Domain.md#auxiliary-image).
 
-A `serverPod` can be defined at the domain scope, which applies to every pod in
-the domain, plus the introspector job's pod, at a specific WebLogic cluster's scope,
-or at a specific WebLogic Server pod's scope.
-Typically, the domain scope is
-the most applicable for the Model in Image use case; for example:
+See [Configuration Example 1: Basic Configuration](#configuration-example-1-basic-configuration).
 
-```
-spec:
-  serverPod:
-    auxiliaryImages:
-    - image: model-in-image:v1
-      imagePullPolicy: IfNotPresent
-      volume: auxiliaryImageVolume1
-```
+#### Source Locations
+Use the optional attributes `sourceModelHome` and `sourceWdtInstallHome` to specify non-default locations of 
```

Comment:
`sourceModelHome` --> `configuration.model.auxiliaryImages[].sourceModelHome` 
ditto for sourceWdtInstallHome

(so that it's obvious these two values are part ofthe aux image stanza)